velox/substrait/proto/substrait/type.proto

```diff
@@ -35,7 +37,13 @@ message Type {
     List list = 27;
     Map map = 28;
 
-    uint32 user_defined_type_reference = 31;
+    UserDefined user_defined = 30;
```

Comment:
Have you copy-pasted the latest version of .proto files from Substrait? Are these the only changes since last update? Are there more changes to other .proto files? It would be nice to update .proto files in a separate PR.

Reply:
The type.proto is the latest version from substrait. 
There still have other changes to other .proto files and we could update them in another PR, your thoughts?

Reply:
@ZJie1 It would be nice to update all .proto files together so that we have a valid snapshot of these.

Reply:
Actually we only need the update of the type.proto, or the test will failed and this pr will be blocked by the PR which will update .proto files. Your opinion? should we wait for all .proto files update?

Reply:
What would it take to update all .proto files? Is this hard?

Reply:
The version bump up will caused code changes in substrait folder. And since there have many changes in substrait since the version we used now,  this work may  not be easy.

Reply:
Sure. Thanks!